app/src/main/java/com/gorbachev/app/EmployeesRepositoryPostgres.java

```diff
@@ -0,0 +1,63 @@
+package com.gorbachev.app;
+
+import com.gorbachev.model.Employee;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ThreadLocalRandom;
+
+public abstract class EmployeesRepositoryPostgres implements EmployeesRepository {
```

Comment:
Почему этот класс абстрактный?
```suggestion
public class EmployeesRepositoryPostgres implements EmployeesRepository {
```